app/bot/__init__.py

```diff
@@ -11,7 +11,35 @@
 _intents.message_content = True
 
 client = discord.Client(intents=_intents)
+conversation_history = []
```

Comment:
We were thinking of having an sqlite database to store the conversation history. Where each channel gets its own context

| channel_id | messages |
| --- | --- |
| 67890 | [{role:user, content:Hello}, {role:assistant, content:Hello to you too}] |
| 12345 | [{role:user, content:bla bla bla}, {role:assistant, content:bla bla bla bla}] |

Reply:
I am not sure about the best way to store the messages

Reply:
The file name of the database would be defined using the [`globalconf.DB_FILE`](https://github.com/btc-raspberrypiclub/piclub-bot/blob/main/app/globalconf.py#L13) variable

Reply:
SQLite would be nice-to-have, but having an in-memory solution is better than nothing. I would mark that as a future upgrade to implement.

Reply:
For supporting channels, using a key-value map, with key = channel_id and value = message array - would make a lot of sense